build/mech_converter.py

```diff
@@ -463,7 +463,15 @@ def convert(input_file, mech_dir, mcm_dir):
             # This code only executes if the break is NOT called, i.e. if the loop runs to completion without the RO2 being
             # found in the species list
             else:
-                ro2_file.write('0 ! error RO2 not in mechanism: ' + ro2List_i + '\n')
+                error_message = ' ****** Warning: RO2 species "' + str(ro2List_i.strip()) + '" NOT found in the mechanism. Please check the RO2 section of your mechansm file for incorrect species names! ******'
```

Comment:
I think this line has snuck into the commit and needs removing.

Reply:
Yep you're right - relic from messing around.

I've removed that line.